sceptre/synapsedev/templates/SynapseCMK-template.json

```diff
@@ -335,12 +303,12 @@
                                             ]
                                         ]
                                     },
-                                    {
-                                        "Fn::ImportValue": "us-east-1-jumpcloud-idp-SynapseDevDeveloperSamlProviderRoleArn"
-                                    },
-                                    {
-                                        "Fn::ImportValue": "us-east-1-jumpcloud-idp-SynapseDevAdminSamlProviderRoleArn"
-                                    }
+                                    "arn:aws:sts::449435941126:assumed-role/AWSReservedSSO_Administrator_693a85eb20cd5043/x.schildwachter@sagebase.org",
```

Comment:
instead of listing every user i believe you should be able to give the admin group access to this key.  The [documentation](https://docs.aws.amazon.com/kms/latest/developerguide/key-policy-overview.html) says..
```
(Required) The principal is the identity that gets the permissions specified in the policy statement.
You can specify AWS accounts, IAM users, IAM roles, and some AWS services as principals in a key
policy. IAM user groups are not a valid principal in any policy type.
```

I think that means you can give the SSO admin role access to this key, so probably all you need here is `arn:aws:sts::449435941126:assumed-role/AWSReservedSSO_Administrator_693a85eb20cd5043`

Reply:
Thx, I'll give it a try (I tried with '/*' but did not think about just the group...).